packages/volto/src/components/manage/Blocks/Teaser/DefaultBody.jsx

```diff
@@ -50,17 +51,28 @@ const TeaserDefaultTemplate = (props) => {
             }
           >
             <div className="teaser-item default">
-              {(href.hasPreviewImage || href.image_field || image) && (
-                <div className="image-wrapper">
-                  <Image
-                    item={image || href}
-                    imageField={image ? image.image_field : href.image_field}
-                    alt=""
-                    loading="lazy"
-                    responsive={true}
-                  />
-                </div>
-              )}
+              {
+                // eslint-disable-next-line
+                url != undefined && !isInternalURL(url) ? (
```

Comment:
@Tishasoumya-02 why not simply url && !isInternalURL? Then you don't need to disable the eslint warning.

Reply:
Yes, I tried doing this, but `blocks-teaser.js` cypress tests keeps failing , not sure why this behavior. I also tried using `isUndefined()` provided by lodash but same results

Reply:
> Yes, I tried doing this, but `blocks-teaser.js` cypress tests keeps failing , not sure why this behavior. I also tried using `isUndefined()` provided by lodash but same results

Works for me locally 
![acceptance-tests-teaser](https://github.com/plone/volto/assets/152852/ed849097-9c07-4429-8b1e-1cb159acd08c)

We also need as part of this fix a cypress test within blocks-teaser that makes use of this preview_image with an external image, maybe it could be an image from demo.plone.org or plone.org @sneridagh what do you suggest for testing this feature of the external image to link to for the teaser block?
An alternative would be an unittest which allows to enter whatever values in the url without actually visiting anything.

Reply:
Okay, not sure why it was not passing on my machine, but made the changes and disabled eslint warning

Reply:
@ichim-david @Tishasoumya-02 We are already using images from the Github CDN in tests, we can use them too:

https://github.com/plone/volto/blob/5f32593d473b0c13823ff81a57f677818f0ccbff/packages/volto/cypress/tests/core/blocks/blocks-chooser.js#L31